app/lib/otis/registration_mover.rb

```diff
@@ -4,10 +4,6 @@ module Otis
   class RegistrationMover
     UMICH_IDP = "https://shibboleth.umich.edu/idp/shibboleth"
```

Comment:
UMICH_IDP could be moved out to some conf file, no?

Reply:
Yes indeed. Will force push a patch.